### PR TITLE
MGMT-2342 move ntp validation from host to cluster validator

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: internal/cluster/validator.go [hunk 1]
```diff
@@ -394,3 +396,40 @@ func (v *clusterValidator) printNetworkPrefixValid(c *clusterPreprocessContext,
 		return fmt.Sprintf("Unexpected status %s", status)
 	}
 }
+
+func (v *clusterValidator) isNtpServerConfigured(c *clusterPreprocessContext) validationStatus {
+	var min int64
```
Comment (ori-amizur) [inline internal/cluster/validator.go:401]: If inventory is missing for a host then this validation should be pending

Comment (tsorya) [reply internal/cluster/validator.go:401]: but for which host? it is not host it is cluster validation

**Comment by ronniel1** [comment]
Please add the Jira ticket to the PR message

**Pull request merged**